src/socket.ts

```diff
@@ -577,63 +580,118 @@ export class DefaultSocket implements Socket {
     });
   }
 
-  addMatchmaker(matchmakerAdd : MatchmakerAdd) : Promise<MatchmakerMatched> {
-    return this.send(matchmakerAdd);
-  }
 
-  createMatch(createMatch: CreateMatch): Promise<Match> {
-    return this.send(createMatch);
+  addMatchmaker(query : string = "*", minCount : number = 2, maxCount : number = 8,  
```

Comment:
I was going off the previous defaults. No strong opinions here other than that I'd be worried about changing a client's existing functionality if they were not previously supplying default params.

Reply:
Correction: The "previous defaults" were actually in the `dotnet-client`. So I'll change this to a default of max 2 because the most common matchmaking scenario is head-to-head.